routes/api/utils.js

```diff
@@ -76,8 +76,4 @@ exports.getTemplates = async function(sort, count, offset, communityId) {
         totalCount: result.count.length,
         templates: result.rows
     };
-};
-
-exports.getMemes = async function(sort, count, offset) {
```

Comment:
I feel like we should make a method like this for community retrieval to avoid duplication. If it's fine with you, I can probably do that along with the 'hot' ranking algo in another PR.

Reply:
@Step7750  Wouldn't it be very easy to add an additional method for getting memes from a community? It's just an extra condition in the where...